rails-commenting.rb

```diff
@@ -54,14 +132,34 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
+     
+
+      # if canceled then 
+      # redirect_to blog_post_path(@post) going back to 
+      # the post allowing then to choose again if they would like to delete
+      # end of method
+      # end of method
       # ---8)
       redirect_to blog_post_path(@post)
     end
   end
 
-  # ---9)
-  private
-  def blog_post_params
+
+
+      # private allows all info for posts to remain private calling on the
+      # define a method called blog_post_params all info in post
+       # ---9)
+          private
+          def blog_post_params
+   
+   
+
+
+    # params require the post and 
+    # the post permits the title and content to be allowed in the data
+    # but must keep what is not called on off the page keeping it private
+    # end of method
+    # end of method
```

Comment:
Above and beyond with the details!

Reply:
Thank you I see now i should have only done the one line, but i liked using it as an exercise. To help myself to read all the pages on ruby and rails while i worked on it , so i could understand it better